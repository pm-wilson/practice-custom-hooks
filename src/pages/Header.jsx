import React from 'react';

const Header = () => {
  return <div><h1>X-files Search</h1></div>;
};

export default Header;
