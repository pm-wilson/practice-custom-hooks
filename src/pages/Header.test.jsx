import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';
import { MemoryRouter } from 'react-router-dom';

describe('Header component', () => {
  it('renders Header', () => {
    const { asFragment } = render(<MemoryRouter><Header /></MemoryRouter>);
    expect(asFragment()).toMatchSnapshot();
  });
});
