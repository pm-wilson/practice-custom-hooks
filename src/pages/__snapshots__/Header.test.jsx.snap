// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders Header 1`] = `
<DocumentFragment>
  <h1>
    X-files Search
    <br />
    <a
      href="/"
    >
      Home
    </a>
  </h1>
</DocumentFragment>
`;
